docs/pages/blog/mui-v5.md

```diff
@@ -0,0 +1,212 @@
+---
+title: MUI v5.0
+description: MUI v5.0 is out 🎉
+date: 2021-09-15T00:00:00.000Z
+authors: ['mbrookes', 'eps1lon', 'mnajdova', 'michaldudak', 'siriwatknp']
+card: true
+---
+
+After over 400 days of development and over 40 canary releases, we are excited to introduce MUI v5.0.0!
+This release features:
+
+- ? depending on timing and strategy [A new branding](#a-new-branding)
+- [High-level goals for v5](#high-level-goals-for-v5)
+- [Improved customizability](#improved-customizability)
+- [New components](#new-components)
+- [A new product line: X](#a-new-product-line-x)
+- [Improved DX](#improved-dx)
+- [v4 migration](#v4-migration)
+- [Design kits](#design-kits)
+- [What's next?](#whats-next)
+
+## A new branding
+
+- I wonder if we shouldn't have a second blog post for this 🤔
```

Comment:
+1 for having a second blog post. Feel strange that the first thing there is on the blog post is the branding. If I come here to see whether I want to use v5, I may not even care for this.

Reply:
I think it is worth mentioning but maybe it is best to do so by the end... And then placing a link to the branding-specific post?!

Reply:
That sounds even better 👍

Reply:
Let's make the customizability efforts and news shine!

Reply:
Done, two blog posts are now in the work.